src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -85,6 +85,10 @@ public string ClientVersion
 
         public bool LegacyCacheCompatibilityEnabled { get; internal set; } = true;
 
+        #region Region
```

Comment:
> #region Region [](start = 7, length = 15)

nit: i thought you didn't like regions...but idk if this is needed

Reply:
region of regions

---
In reply to: [602513676](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2509#discussion_r602513676) [](ancestors = 602513676)